### ISSUE TITLE
Add term tracheal cartilaginous sleeve

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10405]
**Preferred term label:**
Tracheal cartilaginous sleeve

**Synonyms**

**Definition (free text, please give PubMed ID)**
Typical segmentation of the C-shaped tracheal rings is absent and the trachea consists of a long tube of solid cartilage anteriorly and laterally, with a normal membranous tracheal wall posteriorly
PMID: 1554475

**Parent term (use hpo.jax.org/app)**
Anomalous tracheal cartilage HP:0004468

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pfeiffer syndrome (omim101600)
Crouzon syndrome (omim123500)
Apert syndrome (omim101200)
PMID: 1554475

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Already present
[Tracheal cartilaginous sleeve HP:0005347](https://hpo.jax.org/app/browse/term/HP:0005347)
Do you have PMIDs so we can update the annotations?

**Comment by lisannesw** [comment]
@pnrobinson 
"Although the majority of TCS were found in patients with Pfeiffer syndrome, TCS were also identified in children with other FGFR2-associated craniosynostosis syndromes (i.e., Crouzon, Apert) " PMID27228464

Pfeiffer syndrome (omim101600): PMID16633174, PMID15545585, PMID2208766, PMID28468151
Crouzon syndrome (omim123500): PMID12176186, PMID1564116, PMID6546324, PMID28468151
Apert syndrome (omim101200): PMID2385663, PMID1519659, PMID28468151

**Comment by pnrobinson** [comment]
This was HP:0005347
 Tracheal cartilaginous sleeve
Need to add parentage.
